pam/integration-tests/native_test.go

```diff
@@ -223,6 +223,13 @@ func TestNativeAuthenticate(t *testing.T) {
 				PamUser: examplebroker.UserIntegrationMfaWithResetPrefix + "pwquality",
 			},
 		},
+		"Authenticate_user_with_mfa_and_reset_same_password": {
```

Comment:
If you add the same tape to SSH test it should just work too (we symlink most of them in fact).